static/templates/compose.hbs

```diff
@@ -105,21 +105,13 @@
                                 <div class="compose_bottom_bottom_container">
                                     <span id="compose_limit_indicator"></span>
                                     <div class="enter_sends">
-                                        <label id="enter-sends-label" class="compose_checkbox_label tippy-zulip-tooltip" data-tippy-content="Click to switch">
-                                            <span class="enter_sends_true">
-                                                <kbd>Enter</kbd> {{t 'to send' }}
-                                            </span>
-                                            <span class="enter_sends_false">
-                                                <kbd>Ctrl</kbd> + <kbd>Enter</kbd> {{t 'to send' }}
-                                            </span>
-                                            <i class="fa fa-exchange" aria-hidden="true"></i>
-                                            <span class="enter_sends_true">
-                                                <kbd>Ctrl</kbd> + <kbd>Enter</kbd> {{t 'to add a new line' }}
-                                            </span>
-                                            <span class="enter_sends_false">
-                                                <kbd>Enter</kbd> {{t 'to add a new line' }}
-                                            </span>
-                                        </label>
+                                        <span class="enter_sends_true tippy-zulip-tooltip" data-tippy-content="{{t 'Switch behaviour' }}">
```

Comment:
I think this isn't quite the right string; maybe something more like "Configure keyboard controls" or something?  @alya for thoughts.

Reply:
I'm not sure; could be worth discussing in #design. Maybe just "configure"?